app/views/content/train-to-be-a-teacher/how-to-apply-for-teacher-training.md

```diff
@@ -103,17 +103,15 @@ not convinced the admissions team that they’re right for it.
 
 ## Choose your referees
 
-You need 2 references to apply.
+You need 2 referees to apply. You can provide more, but your teacher training provider will only contact 2.
```

Comment:
Do we know how the provider will choose which 2 if you provide more? What's the benefit of providing more?